.github/workflows/workflow-doc.yaml

```diff
@@ -0,0 +1,99 @@
+name: Workflow Doc
```

Comment:
This workflow also deploy packages and Docker image maybe you could find a better name

Reply:
Maybe something like: `Workflow deploy` or `Workflow deploy npm / docker / doc` I prefer the first option

Reply:
I personally like "workflow deploy" or "Deployments"